tectonic-stats-extender/Makefile

```diff
@@ -1,4 +1,85 @@
-.PHONY: vendor
+.PHONY: all build container clean test vendor
```

Comment:
Seems like `build-dirs`, ... are `PHONY` targets as well, right?

I prefer to make the `PHONY` declaration inline, just to make sure not to miss any. Up to you, no strong opinion.

Reply:
I also prefer inline form but kept these up top since it is a template used by k8s. I'll move them inline